From 55273cf6079ddd3b006da69f0113c2c66c03f17e Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Tue, 14 Jul 2020 22:44:22 +0200
Subject: [PATCH] spl: remove dtoc of-pdata generation

Remove the dtoc of-pdata generation. This generation is dependant on
libpython-dev. As OpenWrt does not ship with this dependency, use
pre-generated pdata files and remove the generation from the
build-process.

This only affects RK3328 boards.

Signed-off-by: David Bauer <mail@david-bauer.net>
---
 scripts/Makefile.spl | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/scripts/Makefile.spl
+++ b/scripts/Makefile.spl
@@ -126,7 +126,6 @@ endif
 u-boot-spl-init := $(head-y)
 u-boot-spl-main := $(libs-y)
 ifdef CONFIG_$(SPL_TPL_)OF_PLATDATA
-platdata-hdr := include/generated/dt-structs-gen.h include/generated/dt-decl.h
 platdata-inst := $(obj)/dts/dt-uclass.o $(obj)/dts/dt-device.o
 platdata-noinst := $(obj)/dts/dt-plat.o
 
@@ -355,7 +354,6 @@ $(platdata-hdr) $(u-boot-spl-platdata_c)
 	@# ones around is confusing and it is possible that switching the
 	@# setting again will use the old one instead of regenerating it.
 	@rm -f $(u-boot-spl-all-platdata_c) $(u-boot-spl-all-platdata)
-	$(call if_changed,dtoc)
 
 ifdef CONFIG_SAMSUNG
 ifdef CONFIG_VAR_SIZE_SPL
